GestionareStructuraEchipaDeFormula1/src/gestionareStructuraEchipaDeFormula1/servicii/ScriereInAudit.java

```diff
@@ -0,0 +1,34 @@
+package gestionareStructuraEchipaDeFormula1.servicii;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalTime;
+
+public class ScriereInAudit {
+    private static ScriereInAudit instanta;
+    private static synchronized ScriereInAudit getInstance() {
```

Comment:
not used, pentru ca la fel ca in cazul celorlalte servicii, vrei sa faci singleton, dar ar fi trebuit sa declari constructor privat si sa chemi aceasta metoda unde ai nevoie de instanta (singura) de ScriereInAudit, te rog sa refaci daca doresti